restore/README.md

```diff
@@ -0,0 +1,130 @@
+# Restore action
+
+The restore action restores cache just like the main `cache` action, except it doesn't save the cache. So for cases where users only want to restore cache and not save anything in return, this action is pretty helpful. Moreover, this action accepts the same set of inputs as the `cache` action.
+
+## Inputs
+
+* `path` - A list of files, directories, and wildcard patterns to cache and restore. See [`@actions/glob`](https://github.com/actions/toolkit/tree/main/packages/glob) for supported patterns.
+* `key` - String used while saving cache for restoring the cache
+* `restore-keys` - An ordered list of prefix-matched keys to use for restoring stale cache if no cache hit occurred for key.
+> **Note**
```

Comment:
Do we need this? It is good for user to use same keys but they can use any key they want.
This statement may scare them.

Reply:
Can be removed, existing users would be knowing this already. But I just added this for the new users to know that the key+path combination has to be present in both actions to restore/save the same cache. 

We don't want users to file bugs for these trivial cases due to lack of info. 😅